tests/test_flask.py

```diff
@@ -373,6 +373,21 @@ def test_ical_all(self):
         self.assertEqual(output.data.count('BEGIN:VEVENT'), 15)
         self.assertEqual(output.data.count('END:VEVENT'), 15)
 
+    def test_ical_meeting(self):
+        """ Test the ical_calendar_meeting function. """
+        self.__setup_db()
+
+        output = self.app.get('/ical/calendar/meeting/2/')
+        self.assertEqual(output.status_code, 200)
+        self.assertTrue('BEGIN:VCALENDAR' in output.data)
+        self.assertTrue('SUMMARY:test-meeting2' in output.data)
+        self.assertTrue(
+            'DESCRIPTION:This is another test meeting'
+            in output.data)
+        self.assertTrue('ORGANIZER:pingou' in output.data)
+        self.assertEqual(output.data.count('BEGIN:VEVENT'), 1)
+        self.assertEqual(output.data.count('END:VEVENT'), 1)
```

Comment:
@pypingou there isn't much happening in between ``ical.serialize()`` call in the view function and returning the response. So, even if we dynamically generate the expected output and assert that against the response, it will add little benefit. I believe that the current asserts are good enough. One action, multiple asserts. We don't need to worry about the lines of code in tests.

What do you think?